docs/RELEASE-NOTES.rst

```diff
@@ -14,6 +14,8 @@ Bug Fixes
 `````````
 * :issues:`735` - Deleted rules from routes and ingresses on the same service not cleaned up properly.
 * :issues:`753` - Controller doesn't delete and recreate annotation-based policy rules.
+* :issues:`755` - Controller implements best-match by setting first-match and sorting rules by URI length.
```

Comment:
Outside of the madness that's used for comparing the approot and urlrewrite rules, we're technically sorting in reverse lexical order and not simply URI length.